src/fromager/dependencies.py

```diff
@@ -138,13 +138,16 @@ def default_get_build_backend_dependencies(
     Defaults to result of hook call
     :meth:`~pyproject_hooks.BuildBackendHookCaller.get_requires_for_build_wheel`
     """
+    pbi = ctx.package_build_info(req)
     hook_caller = get_build_backend_hook_caller(
         ctx=ctx,
         req=req,
         build_dir=build_dir,
         override_environ=extra_environ,
     )
-    return hook_caller.get_requires_for_build_wheel()
+    return hook_caller.get_requires_for_build_wheel(
+        config_settings=pbi.config_settings,
```

Comment:
If we're going to always pass it, maybe we can wrap the hook_caller method with a partial? Maybe that's messier internally, though.

Reply:
Sure. I was thinking that the pattern we have is we instantiate the hook caller using our wrapper, and that constructor has all of the args it needs to create the pbi and get the config_settings, but maybe a little repetition isn't so bad for clarity.